charts/cryostat/templates/pvc.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.pvc.enabled }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Release.Name }}
+  {{- $labels := include "cryostat.labels" $ | nindent 4 }}
+  labels: {{ $labels }}
+  {{- with .Values.pvc.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  accessModes: {{ .Values.pvc.accessModes }}
```

Comment:
This one is supposed to be an array, so we probably need the `{{- toYaml . | nindent 4 }}` as in `selector`.